common/coat_of_arms/template_lists/colored_emblem_lists.txt

```diff
@@ -1596,6 +1598,66 @@ colored_emblem_texture_lists = {
 			2000 	= "ce_ogre11.dds"
 			2000 	= "ce_ogre12.dds"
 		}
+		### Harpy
```

Comment:
All these CoAs you added should be in the `charge` list, it's a list used by everyone.